edge-hub/core/src/Microsoft.Azure.Devices.Edge.Hub.Amqp/CbsNode.cs

```diff
@@ -148,8 +148,8 @@ internal static (string token, string audience) ValidateAndParseMessage(string i
 
             try
             {
-                SharedAccessSignature sharedAccessSignature = SharedAccessSignature.Parse(iotHubHostName, token);
-                return (token, sharedAccessSignature.Audience);
+                string audience = message.ApplicationProperties.Map[CbsConstants.PutToken.Audience] as string;
```

Comment:
How does IoTHub do it? 
Is this fully backward compatible?

Reply:
We've checked for the audience properties previously as well (but didn't use it).  It's set by the AMQP library when the SDK sends the authentication token, so I don't see a problem with backward compat.
https://github.com/Azure/azure-amqp/blob/master/src/Cbs/AmqpCbsLink.cs

IoTHub uses the authchain to figure out the connecting vs actor device, and use the API version to prevent compatbility issues.  Since EdgeHub doesn't do the same, this is the next best I could think of.